integration/docker/docker_arch_base.go

```diff
@@ -0,0 +1,12 @@
+// +build !s390x
+
+// Copyright (c) 2019 IBM
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+package docker
```

Comment:
odd, I think the CI is moaning about this, but did not moan about the one in docker_s390x.go?
```
integration/docker/docker_arch_base.go:3:1:warning: package comment should be of the form "Package docker ..." (golint)
The command "bash .ci/static-checks.sh github.com/kata-containers/tests" exited with 1.
```

@sameo - is this maybe at all related to the static check stuff you were looking at yesterday? (that is, not that you broke it, but maybe you know if there is something related to what you were debugging/fixing etc.).

Reply:
It seems the CI wants at least one empty line between the package and the License